### DISCUSSION TITLE
MissionControl Telemetry

### DISCUSSION THREAD

**Discussion by brandflake11** [ndeadly/MissionControl#362] [Q&A] [answered]
Hello,
This might be a silly question, but I was wondering if MissionControl tracks you if you use it. 

> 
If you wish to completely remove all traces of the software ever having been installed (telemetry excepted), you may also want to follow these additional steps

This quote from the readme mentions telemetry. Now, I think this means that it keeps information about your configuration, but I was wondering if MissionControl is tracking users and sending data to a destination online.

Thank you very much for clarifying this.

**Comment by ndeadly** [comment, marked as answer]
Mission control doesn't record any telemetry, the line from the readme you quote is just indicating that it's possible that Nintendo's telemetry records information on paired controllers and that simply deleting the module components from your SD card and clearing the pairing database will not remove this information. That said, as far as I know Atmosphere blocks all telemetry from being sent to Nintendo these days anyway so it's not really something you need to worry about.

**Reply by brandflake11** [reply]
Thanks so much for taking the time to clarify this. This makes more sense.
